docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     image: localstack/localstack:0.11.2
     networks:
       - hmpps
-    container_name: localstack
+    container_name: hmpps-tier-to-delius-update-localstack
```

Comment:
to avoid clashes with any stopped containers from other projects when running locally